parts/k8s/kubernetesmastercustomdata.yml

```diff
@@ -386,7 +386,6 @@ runcmd:
 - systemctl restart docker
 - mkdir -p /etc/kubernetes/manifests
 - usermod -aG docker {{WrapAsVariable "username"}}
-- /usr/lib/apt/apt.systemd.daily
```

Comment:
why it's removed?

Reply:
this part can take more than 5 mins to finish on a Standard_F1 sku. This makes the CSE failing rate very high and in turn failing the template deployment a lot. The latest change in ACS-Engine moved this part to cluster-provision.sh. I just copied that change and it solved the F1 sku problem.